tests/foreman/api/test_setting.py

```diff
@@ -1,13 +1,13 @@
 # -*- encoding: utf-8 -*-
 """Test class for Setting Parameter values
 
-:Requirement: Setting
+:Requirement: Test Cases for component Settings
 
 :CaseAutomation: Automated
 
 :CaseLevel: Acceptance
 
-:CaseComponent: API
+:CaseComponent: Settings
```

Comment:
```suggestion
:CaseComponent: API
```
AFAIK this is still under discussion and we could keep as it is ..

Reply:
Aligning CaseComponent with bugzilla components has one unintended consequence that was discovered recently and is being discussed, but I highly doubt it will stop the whole effort - we are much more likely to introduce another field to capture data we are missing, or just ignore the issue. Our management was very clear that they want to use CaseComponent for finding component owner, and it means that "API" value is usually not what we want.

Reply:
@mirzal  thanks to clear on then it's good to keep the new change .